client/state/reader/onboarding/handlers.ts

```diff
@@ -40,11 +40,7 @@ export const dispatchProfileCompleteNotice = ( store: Store, action: UnknownActi
 			...state.userSettings.settings,
 			...( typeof action.settingValues === 'object' ? action.settingValues : {} ),
 		};
-		const gravatarDetails = {
-			...state.gravatarStatus,
-			gravatarDetails: { has_gravatar: !! state.gravatarStatus.tempImage },
```

Comment:
We may not have a `tempImage`, and we don't want to overwrite `true` if the avatar has been added.